src/v1/components/surfaces/FloatingPanel/__tests__/useFloatingPanelsContext.test.ts

```diff
@@ -0,0 +1,26 @@
+import { renderHook } from "@testing-library/react";
+import useFloatingPanelContext from "../useFloatingPanelContext";
+
+describe("useFloatingPanels hook", () => {
+  test("throws error when it's not used inside Floating.Provider", () => {
+    // Temporarily suppress console.error
+    const originalConsoleError = console.error;
+    console.error = jest.fn();
```

Comment:
What I would do - which I suspect is more robust if we parallelised the tests.

I would do
````
const spies = {
  console: {
    error: jest.spyOn(console, 'error').mockImplementation(() => {})
    }
}

describe('blah', () => {
  beforeEach(() => {
    spies.console.error.resetMocks(); //. ? or is it clearMocks I cant remember
  })
  it('blah', () => {
  })
  ```

Reply:
feel free to resolve once read

Reply:
I don't think I fully understand but I will talk to you about it in a call